x-pack/plugins/lens/public/embeddable/embeddable.tsx

```diff
@@ -323,7 +325,15 @@ export class Embeddable
     if (this.input.onLoad) {
       this.input.onLoad(true);
     }
+    const executionContext = this.deps.executionContext.create({
+      type: this.savedVis.type ?? 'lens',
+      name: this.savedVis.visualizationType ?? '',
+      description: this.savedVis.title ?? this.savedVis.description ?? '',
+      id: this.id,
+      url: this.output.editUrl,
```

Comment:
I saw `url: "app/lens#/edit_by_value"` during the testing. What url should I use instead?

Reply:
that's probably correct url, other state is passed thru navigateToAppwithstate

Reply:
For the "by-value" Lens panels we can't provide a linkable URL yet. Is there a way we could link to the dashboard URL instead, as the dashboard is has a URL?

Reply:
>  Is there a way we could link to the dashboard URL instead, as the dashboard is has a URL?

not yet. in https://github.com/elastic/kibana/issues/102626 I'm going to add nested context support, so we can create and link `dashboard context` and `visualization context`.